commitizen/git.py

```diff
@@ -92,6 +92,11 @@ def tag(tag: str, annotated: bool = False, signed: bool = False) -> cmd.Command:
     return c
 
 
+def add() -> cmd.Command:
+    c = cmd.run("git add .")
```

Comment:
I don't think this is what `-a` does. In this case even non-tracked files are being added. `-a` should happen on `git commit -a`

Reply:
Thank you! I misunderstood what git commit -a usage. I will modify this PR.